About.html

```diff
@@ -114,33 +114,30 @@ <h3 class="partner-heading">Partner</h3>
       <hr class="heading-underline" />
       <div class="partners-list">
         <div class="img-car">
-        <img src="assets/images/partners/mozilla 1.png" alt="" class="p-image" />
-      
-        <img src="assets/images/partners/googlekorea 1.png" alt="" class="p-image" />
-      
-        <img src="assets/images/partners/naver 1.png" alt="" class="p-image" />
-      </div>
+          <img src="assets/images/partners/mozilla 1.png" alt="" class="p-image" />
+          <img src="assets/images/partners/googlekorea 1.png" alt="" class="p-image" />
+          <img src="assets/images/partners/naver 1.png" alt="" class="p-image" />
+        </div>
         <div class="img-car2">
-        <img src="assets/images/partners/daumkakaologo 1.png" alt="" class="p-image" />
-      
-        <img src="assets/images/partners/airbnb 1.png" alt="" class="p-image" />
-      </div>
+          <img src="assets/images/partners/daumkakaologo 1.png" alt="" class="p-image" />
+          <img src="assets/images/partners/airbnb 1.png" alt="" class="p-image" />
+        </div>
```

Comment:
Great design @maths29 🤝
- [ ] I would also recommend increasing the font size of the main heading of the about section so that your design can replicate the design provided.